src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig

```diff
@@ -406,3 +406,21 @@ td.main, td.menu {
 .collapsed-menu #minimizePanelIcon {
     transform: rotate(180deg);
 }
+
+.sf-icon {
+    margin-right: 5px;
+    vertical-align: middle;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+}
+.sf-icon.sf-2x {
```

Comment:
I would rather call it ``sf-icon-large`` than ``sf-2x``. Last time I computed ``2 * 16px``, I got ``32px``, not ``24px``, so the name is currently lying